kolibri/core/apps.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import unicode_literals
+
+from django.apps import AppConfig
+
+
+class KolibriCoreConfig(AppConfig):
+    name = 'kolibri.core'
+
+    def ready(self):
+        from django.db import connection
+
+        # Only run the PRAGMAs if we're using SQLite
+        if connection.vendor == "sqlite":
+            cursor = connection.cursor()
+            cursor.execute("PRAGMA synchronous=OFF;")
```

Comment:
'OFF' should still be faster - but from the SQLite docs, 'NORMAL' should give us some efficiencies, while not being as dangerous.